content/en/docs/contributing/release-process.md

```diff
@@ -48,7 +47,7 @@ following:
    Step 4 under "Prerequisites" on the [release-process][1] page means I need
    access to the [cert-manager-release][2] project on GCP.
 
-   I would need to be an "Editor" on that project. More specifically, the roles
+   I need to be an "Editor" on this project. More specifically, the roles I need are:
    that I will be using are:
```

Comment:
This line needs removing

Reply:
Specifically, "that I will be using are: "

Reply:
Thanks! Fixed in 5e1bc1d